rust-peer/fly.toml

```diff
@@ -0,0 +1,28 @@
+# fly.toml app configuration file generated for universal-connectivity-rust-peer on 2023-09-24T16:20:25+10:00
+#
+# See https://fly.io/docs/reference/configuration/ for information about how to use this file.
+#
+
+app = "universal-connectivity-rust-peer"
+primary_region = "syd"
```

Comment:
Any thoughts on moving this to Europe/US, where most users will likely be connecting from?

Reply:
Happy to, this is just the default 😅